### ISSUE TITLE
Failed in m1n1dumpIt

### ISSUE THREAD

**Issue by nichnaib** [D1rkMtr/DumpThatLSASS#3]
Hello,
Thanks for the tool but, It fails with this error:
````
[0xe8] Process: 424 C:\Windows\System32\lsass.exe
[0x11c] Process: 472 C:\Windows\System32\lsass.exe
[0x48c] Process: 580 C:\Windows\System32\lsass.exe
[0x4b4] Process: 580 C:\Windows\System32\lsass.exe
[0x4c0] Process: 580 C:\Windows\System32\lsass.exe
[0x70c] Process: 580 C:\Windows\System32\lsass.exe
[0x16bc] Process: 580 C:\Windows\System32\lsass.exe
Failed in m1n1dumpIt:M1n1DumpWr1teDump (32)
````
It can recover the PID of the lsass.exe process and it can create the file, so maybe the error is somewhere inside MiniDumpWriteDump function? GetLastError returns 32.

I've tried it both on a Windows Server 2012 R2 Standard 9600 x64 and on a Windows Server 2008 R2 Standard 7601 Service Pack 1 x64, same error occurs.
I've commented the sandbox evasion as you suggested. 

Any idea on what can it be?

**Comment by D1rkMtr** [comment]
![image](https://user-images.githubusercontent.com/110354855/202518428-07217afc-85c5-45b9-a278-89a18c064b73.png)

it works fine in win10 , not try it on win server

**Comment by D1rkMtr** [comment]
also work fine in win server 19

![image](https://user-images.githubusercontent.com/110354855/202520327-78234ed9-da25-41cd-9443-225f6b201b1c.png)


i updated the code re-check it if it help

**Comment by D1rkMtr** [comment]
it will not work if ppl is enabled : lsass is a protected process .

**Comment by nichnaib** [comment]
Thanks. RunAsPPL is 0 but i suppose the version of the server is too old

**Comment by nichnaib** [comment]
it seems that sophos enpoint agent does not detect the executable (or it does but only shows it in the cloud's dashboard and not locally on the computer) but it fails with the same error and different error code:
![image](https://user-images.githubusercontent.com/26966145/202666818-007ba98a-b611-4a08-8b95-de96c05155ab.png)
If sophos endpoint agent is disabled, dump gets created correctly (win server 19) 
So basically sophos doesnt detect it but does not let you dump anything either.
